### ISSUE TITLE
Require to fetch participants info, chat info and meeting info for storing data in my app.

### ISSUE THREAD

**Issue by Meri994** [saibotma/jitsi_meet_wrapper#16]
@saibotma Thanks for the library. I have checked in and it's working on Pixel XL API 31. It has saved me lots of time. Thanks again.

I am also getting logs for when any participants join, left, or any message arrives in the meeting.
Here are the logs for that:
```
D/JitsiMeetSDK( 7218): ExternalAPI Sending event: PARTICIPANT_LEFT with data: { NativeMap: {"participantId":"515e6b41"} }
D/JitsiMeetSDK( 7218): ExternalAPI Sending event: CHAT_MESSAGE_RECEIVED with data: { NativeMap: {"senderId":"83409ad1","isPrivate":false,"message":"yes please"} }
D/JitsiMeetSDK( 7218): ExternalAPI Sending event: PARTICIPANT_JOINED with data: { NativeMap: {"name":"tester meri","participantId":"cbc8a823","role":"moderator"} }
```
Are there any methods available for getting all these longs in your flutter example project? I like to store meetings and participants' data in my application. It will be great if you find it. Thank you.

You have mentioned in the GitLab repo that currently more options are not available same as ios and android in a flutter. Can you add in this repo?

**Comment by Meri994** [comment]
There are few methods available for check join/terminal conference.

```
JitsiMeetingListener(
        onOpened: () => debugPrint("onOpened"),
        onConferenceWillJoin: (url) => debugPrint("onConferenceWillJoin: url: $url"),
        onConferenceJoined: (url) => debugPrint("onConferenceJoined: url: $url"),
        onConferenceTerminated: (url, error) {
          debugPrint("onConferenceTerminated: url: $url, error: $error");
        },
        onClosed: () => debugPrint("onClosed"),
      ),
```

      I like to get the same delegate methods which are available in ios SDK. My requirement is to get participants' info and store it locally.
      
```
     participantJoined
     participantLeft
     chatMessageReceived
 ```

**Comment by saibotma** [comment]
Now I get it. I can't guarantee that I will implement this soon. However, I'll leave this issue open, and I am happy for PRs.

**Comment by Meri994** [comment]
> Now I get it. I can't guarantee that I will implement this soon. However, I'll leave this issue open, and I am happy for PRs.

Actually, I am new in a flutter. If possible can you do it else guide me I will do that? I have checked jitsi_meet_wrapper_platform_interface and you have added  jitsi_meeting_listener.dart file. In that, I can see

```
  JitsiMeetingListener({
    this.onOpened,
    this.onConferenceWillJoin,
    this.onConferenceJoined,
    this.onConferenceTerminated,
    this.participantJoined,
    this.onClosed,
  });
```
What steps I need to do for adding more methods in this listener?

**Comment by Meri994** [comment]
Hey, the app is crashing in the android 12 google pixel device. When I click on join the meeting, the app is crash but it is properly working in Pixel XL API 31 enumator. Have you checked in android 12 supported phone? Thanks

**Comment by Meri994** [comment]
@saibotma After creating the apk, the app is crashed on all android devices. I have checked in android 9, android 10, android 11, and android 12 supported devices. Kindly look into it and do needful. Waiting for your response.

**Comment by saibotma** [comment]
Hey @Meri994, sorry for the late response. I am very busy at the moment. I will try to check on an Android 12 phone this week.

**Comment by Meri994** [comment]
> Hey @Meri994, sorry for the late response. I am very busy at the moment. I will try to check on an Android 12 phone this week.

Thanks

**Comment by Meri994** [comment]
> > Hey @Meri994, sorry for the late response. I am very busy at the moment. I will try to check on an Android 12 phone this week.
> 
> Thanks

@saibotma Hey,  I forgot to add proguard file, so it was crashing. I have added that file and then check with the android 12 device, It's working properly. Thanks for your support.

I require to get above mention methods, When you get chance, please add those methods. Thank you.

**Comment by saibotma** [comment]
Implemented in 0.0.4.
